File: src/W02/conditions.c
/**
 * conditions.c
 *
 * Josh Kaplan
 * jk@ucf.edu
 *
 * Here is an example of if/else conditions in C.
 */

#include <stdio.h>

int main()
{
    int x = 10;

    if (x % 2 == 0)
    {
        printf("You entered an even number!\n");
    }
    else if (x < 10)
    {
        printf("x is less than 10!\n");
    }
    else if (x > 10 && x < 20)
    {
        printf("x between 10 and 20!\n");
    }
    else
    {
        printf("You entered an odd number!\n");
    }
}

File: src/W02/conditions.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""

conditions.py

Josh Kaplan
jk@ucf.edu

Here is an example of if/elif/else conditions in Python.

"""

x = 10

if x % 2 == 0:
    print('You entered an even number!')
elif x < 10:
    print('x is less than 10!')
elif x > 10 and x < 20:
    print('x between 10 and 20!')
else:
    print('You entered an odd number!')

